Kiosk/App/Networking/ArtsyAPI.swift

```diff
@@ -303,6 +307,9 @@ extension ArtsyAPI : MoyaTarget {
 
         case MyBidPositionsForAuctionArtwork:
             return stubbedResponse("MyBidPositionsForAuctionArtwork")
+            
+        case MyBidPosition:
+            return stubbedResponse("MyBidPosition")
```

Comment:
There's a directory with other `.json` files that needs to have a file in it that is named `MyBidPosition.json`, which contains a stubbed network response, otherwise the tests will crash.